build/fr24feed.sh

```diff
@@ -5,6 +5,15 @@ case $arch in
     url=https://repo-feed.flightradar24.com/rpi_binaries/fr24feed_${FR24FEED_ARMHF_VERSION}_armhf.tgz # force version 1.0.25-3 because of broken version for rpi
     dirname=fr24feed_armhf
     ;;
+  arm64)
+    url=https://repo-feed.flightradar24.com/rpi_binaries/fr24feed_${FR24FEED_ARMHF_VERSION}_armhf.tgz # force version 1.0.25-3 because of broken version for rpi
+    dirname=fr24feed_armhf
+    dpkg --add-architecture armhf
+    apt update
+    apt dist-upgrade
+    apt install libc6:armhf libstdc++6:armhf libusb-1.0-0:armhf
```

Comment:
Could you also remove cache to have a smaller image ?

```
apt-get install -y \
    xxx && \
    rm -rf /var/lib/apt/lists/*
```

Reply:
Done